base9/Settings.cpp

```diff
@@ -521,11 +518,10 @@ template<typename T> T Settings::getOrRequest (Node & n, string && f)
     {
         string input = "";
         cout << "Please enter your desired setting for field: '" + f + "':\n";
-        getline(cin, input);
-        cout << "Field '" + f + "' set to: " << input << endl;
```

Comment:
Wait, I already did nix it.  You're looking at a previous commit.
https://github.com/maxvonhippel/base/blob/master/base9/Settings.cpp